Microsoft.Research/RegressionTest/ClousotTests/v.cs

```diff
@@ -216,6 +1728,18 @@ public void Slice2SequentialFromSourcesV40()
         TestDriver.BuildAndAnalyze2Slicing(options);
     }
 
+    [Theory]
+    [MemberData("ClousotTestInputs")]
+    public void XunitSlice2SequentialFromSourcesV40(int optionsIndex)
```

Comment:
:memo: We create one `[Theory]` method from each of the existing test methods that references **ClousotTestInputs.xml**, and xUnit.net will automatically break up each theory into 175 (currently) discreet unit tests.